packages/dev/core/src/Materials/Node/Blocks/Fragment/shadowMapBlock.ts

```diff
@@ -4,6 +4,7 @@ import type { NodeMaterialBuildState } from "../../nodeMaterialBuildState";
 import { NodeMaterialBlockTargets } from "../../Enums/nodeMaterialBlockTargets";
 import type { NodeMaterialConnectionPoint } from "../../nodeMaterialBlockConnectionPoint";
 import { RegisterClass } from "../../../../Misc/typeStore";
+import { ShaderLanguage } from "core/Materials/shaderLanguage";

```

Comment:
should it not be ../../../../ instead of core for consistency ?

Reply:
I think they are the same now @RaananW ?

Reply:
yes they are. Just cosmetic